go/private/actions/link.bzl

```diff
@@ -241,41 +212,3 @@ def _extract_extldflags(gc_linkopts, extldflags):
         else:
             filtered_gc_linkopts.append(opt)
     return filtered_gc_linkopts, extldflags
-
-def _check_conflicts(arcs):
-    importmap_to_label = {}
-    for arc in arcs:
-        if arc.importmap in importmap_to_label:
-            return """package conflict error: {}: multiple copies of package passed to linker:
-	{}
-	{}
-Set "importmap" to different paths or use 'bazel cquery' to ensure only one
-package with this path is linked.""".format(
-                arc.importmap,
-                importmap_to_label[arc.importmap],
-                arc.label,
-            )
-        importmap_to_label[arc.importmap] = arc.label
-    for arc in arcs:
-        for dep_importmap, dep_label in zip(arc._dep_importmaps, arc._dep_labels):
```

Comment:
I think we'll just get linker errors for these? seems ok?

Reply:
Can we reproduce this case in an example or even a test?